src/pods/test-sentences/components/body.component.tsx

```diff
@@ -1,77 +1,202 @@
+  setVerbsForms: (string) => void;
+  setShowSetenceResult: (boolean) => void;
+}
+
+const TensesButtonGroup: React.FunctionComponent<ButtonGroupProps> = props => {
+  const {
+    sentenceSelected,
+    setRightAnswerValue,
+    setVerbsForms,
+    setShowSetenceResult,
+  } = props;
+  const { present, past, participle, rightTenseAnswer } = sentenceSelected;
+
+  const handleButtonValue = e => {
+    e.currentTarget.value === rightTenseAnswer
+      ? setRightAnswerValue(QuestionStatus.correct)
+      : setRightAnswerValue(QuestionStatus.incorrect);
+    setVerbsForms(e.currentTarget.value);
+    setShowSetenceResult(true);
+  };
+  return (
+    <ButtonGroup
+      variant="contained"
+      color="primary"
+      size="large"
+      aria-label="contained primary button group"
+    >
+      <Button onClick={handleButtonValue} value="Present">
+        {present}
+      </Button>
+      <Button onClick={handleButtonValue} value="Past">
+        {past}
+      </Button>
+      <Button onClick={handleButtonValue} value="Participle">
+        {participle}
+      </Button>
+    </ButtonGroup>
+  );
+};
+
+interface ShowResultsProps {
+  sentenceSelected: SentenceEntityVm;
+  rightAnswerValue: number;
+}
+
+const ShowResultsSentence: React.FunctionComponent<ShowResultsProps> = props => {
+  const { sentenceSelected, rightAnswerValue } = props;
+  const { rightTextAnswer, prefixSentence, sufixSentence } = sentenceSelected;
+
+  const {
+    answer,
+    ballons,
+    buttonWrong,
+    buttonRight,
+    correctSpanStyle,
+    insideBtn,
+    insideRightAnswer,
+    picture,
+    verbsForm,
+  } = styles;
+
+  return (
+    <>
+      {(() => {
+        switch (rightAnswerValue) {
+          case QuestionStatus.correct:
+            return (
+              <div className={insideRightAnswer}>
+                <div className={buttonRight}>
+                  <span>RIGHT !!!!</span>
+                </div>
+                <div>
+                  <img
+                    className={`${picture} ${ballons}`}
+                    src={`/assets/right-answer/right.png`}
+                    alt=""
+                  />
+                </div>
+              </div>
+            );
+          case QuestionStatus.incorrect:
+            return (
+              <div className={insideRightAnswer}>
+                <div>
+                  <div className={buttonWrong}>
+                    <div className={insideBtn}>
+                      <span>Oops... nope</span>
+                    </div>
+                  </div>
+                  <span className={answer}>Answer</span>
+                  <span className={verbsForm}>
+                    <span>{prefixSentence}</span>{' '}
+                    <span className={correctSpanStyle}>{rightTextAnswer}</span>{' '}
+                    <span>{sufixSentence}</span>
+                  </span>
+                </div>
+              </div>
+            );
+          default:
+            return null;
+        }
+      })()}
```

Comment:
We need to componentize this, something like, create subcomponents (single responsiblity principle):

```tsx
<main className={mainContainer}>
  <div>
    <h1 className={title}>{translation.toUpperCase()}</h1>
    <VerbPicture src={`/assets/verb-images/buy.png`}/>
    { (showSentenceResult) ?
       <QuestionComponent/>
      :
        <ResultComponent/>
    }
  </div>
  {showSentenceResult && (
      <NextVerbButton/>
  )}
</main>
```

Reply:
Done.